docs/assets/diagrams/transfer/03-confirm-payee.puml

```diff
@@ -0,0 +1,118 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+Alice -> App: Initiate this transaction
+
+rnote right of App #Light
+  ""firebase.firestore()""
+  ""  .collection("transactions")""
+  ""  .doc("456")""
+  ""  .set({""
+  ""    sourceAccountId: "bob.fspb",""
+  ""    consentId: "123",""
+  ""    amount: {""
+  ""      amount: "20",""
+  ""      currency: "USD"""
+  ""    },""
+  ""    payer: {""
+  ""      personalInfo: {""
+  ""        complexName: {""
+  ""          firstName: "Bob",""
+  ""          lastName: "Beaver"""
+  ""        }""
+  ""      },""
+  ""      partyIdInfo: {""
+  ""        partyIdType: "MSISDN",""
+  ""        partyIdentifier: "+1-111-111-1111",""
+  ""        fspId: "fspb",""
+  ""      }""
+  ""    },""
+  ""  }, { merge: true })""
+end rnote
+
+App -> Firebase ++: Create a new transaction document
+Firebase <-- App: ""200 OK""
+
+rnote right of Firebase #Light
+  ""event(""
+  ""  "document.create",""
+  ""  "transactions/456",""
```

Comment:
this is not created under transactions/users/*, is this intended?

Reply:
Yes, this is based on [JJ's suggestion](https://github.com/mojaloop/pisp-demo-server/pull/4#pullrequestreview-443556058) to store everything under `transactions` collection so that the server could just listen to `transactions`. 

And then, the user will listen to all documents with field `userId` matching the id of the user based on Firebase auth.